F2_Operações de Crédito - Empréstimos (Loans)_Operações de Crédito - Empréstimos (Loans)/get-contracts-{contractId}-scheduled-instalments/get-contracts-contractId-scheduled-instalments-24.1.json

```diff
@@ -0,0 +1,26 @@
+{
+    "data": {
+        "typeNumberOfInstalments": "SEMANA",
+        "totalNumberOfInstalments": 78,
+        "typeContractRemaining": "SEMANA",
+        "contractRemainingNumber": 74,
+        "paidInstalments": 4,
+        "dueInstalments": 3,
+        "pastDueInstalments": 2,
+        "balloonPayments": [
+            {
+                "dueDate": "NA",
+                "currency": "NA",
+                "amount": 0.0000
+            }
+        ]
+    },
+    "links": {
+        "self": "https://api.caixa.gov.br/open-banking/loans/v1/contracts/befb174b-b96f-381e-9150-effa8dc43c7e/scheduled-instalments"
+    },
+    "meta": {
+        "totalRecords": 1,
+        "totalPages": 1,
+        "requestDateTime": "2021-10-02T18:10:33Z"
```

Comment:
A data de consulta está bem além das datas de início e fim do contrato. Considerando esse cenário, com essas datas, o que achas de modificar os campos `dueInstalments` e `pastDueInstalments` para `"dueInstalments": 0` e `pastDueInstalments: 74`?

Reply:
Eu acho que nao vai funcionar. Ela teria uma divida no nome dela. E essa persona tem varios financiamentos depois desse emprestimo. Eu vou alterar as datas para algo mais recente